cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -160,6 +165,15 @@ class RadixTree
     RadixTreeNode<T> * get_current_node() const { return m_current_node; }
     key_type get_unique_node() const { return m_unique_id; }
 
+    class CallProfilerPK
+    {
+    private:
+        CallProfilerPK() = default;
+        friend CallProfilerSerializer;
+    };
+
+    const std::unordered_map<std::string, key_type> & get_id_map(CallProfilerPK const &) const { return m_id_map; }
```

Comment:
In order to expose the id_map to the serializer, I add the passkey in CallProfiler.